src/gr11review/part2/Utility.java

```diff
@@ -1,9 +1,357 @@
 package gr11review.part2;
-import java.io.*;
 
+import java.io.*;
+import java.util.Scanner;
 public class Utility {
 
+  
+/**
+* This method zipZap lets the user specify a String
+* and removes the middle character within a 3 letter string in the pattern
+* z_p, wherever this is found
+*
+* @param str  The String which the program searches for instances of 'z_p'
+* @return A string with the middle character removed from instances of 'z_p'
+* @author Natalia Garcia-Arias
+*/
+    public static String zipZap(String str){
+
+        String strNew;
+
+        //Removes all letters between the pattern "z_p"
+        strNew = str.replaceAll("z[a-zA-Z]p", "zp");
+        
+        return strNew;
+    } 
+
+ // Methods 2
```

Comment:
@alyannasantos all methods should have a javadoc comment to describe method purpose, parameter, return value, and author